File: Python Word examples/field/deleteField.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.fields_api.FieldsApi(configuration)

file_name = "Fields.docx"
paragraph_path = "sections/0/paragraphs/0"
index = 0
extra = {'folder': 'input', 'dest_file_path': 'output/DeleteField.docx'}
api.delete_field(file_name, paragraph_path, index, **extra)

File: JavaScript Word Examples/FieldsApiDemo/deleteField.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.FieldsApi(apiClient);
    
    var fileName = "Fields.docx";
    var paragraph_path = "Section/0/Body/0/Paragraph/0";
    var destFilePath = "output/Fields/DeleteField_1.docx";
    var index = 0;
    var opts = { 'folder': 'Fields' };
    instance.deleteField(fileName, paragraph_path, index, destFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();